src/auth-service/services/mailer.js

```diff
@@ -272,6 +272,54 @@ const mailer = {
       };
     }
   },
+
+  feedback: async ({ email, message, subject } = {}) => {
+    try {
+      let bcc = constants.REQUEST_ACCESS_EMAILS;
+
+      const mailOptions = {
+        from: {
```

Comment:
Been able to address our email address name issue from this new PR. Was "lucky" this time! :) `So I guess I will have to compare and perhaps "merge content from the other older PR. @FaithDaka 
<img width="532" alt="Screenshot 2022-10-11 at 21 13 56" src="https://user-images.githubusercontent.com/1590213/195168677-a00133d1-4534-4923-a67e-29b1f81f41cf.png">